src/components/AuthorizeURLBuilder.tsx

```diff
@@ -207,6 +263,46 @@ export default function AuthorizeURLBuilder() {
           </div>
         ))}
       </div>
+
+      {options.availableScopes.length > 0 ? (
+        <div>
+          <label>
+            scopes
+          </label>
+          {options.availableScopes.map(scope => (
+            <label className="text-gray-700 text-sm block my-2">
+                <input
+                  type="checkbox"
+                  id={scope}
+                  className="mr-2"
+                  checked={options.selectedScopes.includes(scope)}
+                  onChange={() => toggleScope(scope)}
+                />
+                {scope}
+            </label>
+          ))}
+
+          <div className="mb-4 grid grid-cols-2 gap-4">
+            <div>
+              <label className="block text-gray-700 text-sm font-bold mb-2" htmlFor="scopes_quirk">
+                scopes quirk handling
+              </label>
+              <select className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
+                id="scopes_quirk"
+                placeholder="scopes quirk handling"
+                value={options.scopes_quirk}
+                onChange={(event) => updateOption('scopes_quirk', event)}
+              >
+                <option value="none">none</option>
+                <option value="login_hint">login_hint</option>
+              </select>
+              <small>
+                Some integrations, like Auth0, require that you pass scopes through the login_hint.<br />
+              </small>
+            </div>
+          </div>
+        </div>
+      ) : null}
       
       {options.acr_values.length == 1 ? (
         <div className="mb-4 grid grid-cols-2 gap-4">
```

Comment:
scopes, scopes quirk handling and acr values handling should probably all be in the same grid so they can take up some horizontal space